plugins/Goals/lang/en.json

```diff
@@ -68,30 +68,30 @@
         "Filename": "filename",
         "GoalConversion": "Goal conversion",
         "GoalConversions": "Goal conversions",
-        "GoalConversionsBy": "Goal %s conversions by type of visit",
-        "GoalIsTriggered": "Goal is triggered",
-        "GoalIsTriggeredWhen": "Goal is triggered when",
+        "GoalConversionsBy": "%s goal conversions by type of visit",
+        "GoalIsTriggered": "A goal is triggered",
+        "GoalIsTriggeredWhen": "A goal is triggered when",
```

Comment:
I don't the those changes make sense in the context where the translations are used.
```suggestion
        "GoalIsTriggered": "Goal is triggered",
        "GoalIsTriggeredWhen": "Goal is triggered when",
```

Reply:
There should be an article there. It is either "A goal is triggered", or "The goal is triggered".
Alternatively, it could be "Goal triggered" and "Goal triggered when".
Either "a/the" and "is", or nothing.